loadimpact/loadimpact.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+// An example of usign Pantheon's Quicksilver technology to do 
+// a performance test using Load Impact
+
+// Provide the API Key provided by Load Impact
+// For extra security, you can store this information in
+// the private area of the files directory as documented
+// at https://github.com/pantheon-systems/quicksilver-examples.
+$api_key = 'add-api-key-here';
```

Comment:
I think we should standardize on an abstraction layer. Part of my reasoning is for the security benefit. While some of our quicksilver examples involve keys that aren't terribly sensitive I think we should just stick with the best practice of "don't commit keys to any repo" so as limit confusion.

Just as important for me is clearing a path toward treating QS scripts as modules/plugins as opposed to custom code. Right now these scripts are examples meant to be edited by the implementor. There should not be an expectation (right now) that a Slack notification script in one Pantheon site is identical to a Slack notification script on another Pantheon site.

I anticipate that expectation will evolve. Our user base of Drupal and WordPress developers is accustomed to running commands like `drush dl modulename` and treating community code as not-to-be-edit (hacking core kills kittens and all that). With `terminus quicksilver install examplename` there is a road to treating QS code like modules to which configuration is passed.

Of course Quicksilver as a whole is still evolving. And maybe for the stage we are at now we should be actively fighting (possibly) premature abstraction. If we want Quicksilver users to think "this is code I need to read, understand and edit to my use case" then let's continue to include pieces like this that require editing the php file. If we want QS users to think "I install an example and it just works" then we need abstractions like key management.